lib/bulk_data_test_kit/v2.0.0/client/groups/delete.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require_relative '../export_types'
+require_relative '../urls'
+
+require_relative '../tests/kick_off'
+require_relative '../tests/delete'
+
+module BulkDataTestKit
+  module BulkDataV200
+    module Client
+      module Groups
```

Comment:
Not sure if this is something that should be changed, but I am thinking to be consistent with the rest of the groups in other bulk data test kit suites, there is no need to add the Groups module here

Reply:
I'm good with keeping things consistent. I've made this change.